adhocracy4/comments/static/comments/Comment.jsx

```diff
@@ -8,12 +8,15 @@ var CommentManageDropdown = require('./CommentManageDropdown')
 
 var React = require('react')
 var django = require('django')
-var moment = require('moment')
 
 var safeHtml = function (text) {
   return { __html: text }
 }
 
+var localeDate = function (dateStr) {
+  return new Date(dateStr).toLocaleString(document.documentElement.lang)
```

Comment:
I think MDN is wrong here:

> ECMAScript defines a string interchange format for date-times **based upon a simplification of** the ISO 8601 Extended Format. The format is as follows: YYYY-MM-DDTHH:mm:ss.sssZ
> -- https://www.ecma-international.org/ecma-262/5.1/#sec-15.9.1.15

Our format uses a space instead of `T`. This is valid ISO 8601, but is not supported by ECMAScript.